content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -60,7 +60,7 @@ It acts synchronously to modify pods as they are created or updated. When this p
   1. It adds a `volumeSource` to each container of the pod mounted at `/var/run/secrets/kubernetes.io/serviceaccount`.
 
 #### Bound Service Account Token Volume
-{{< feature-state for_k8s_version="v1.13" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.21" state="beta" >}}
 
 When the `BoundServiceAccountTokenVolume` feature gate is enabled, the service account admission controller will
```

Comment:
We're a bit more strict about beta docs, so I recommend:
```suggestion
When the `BoundServiceAccountTokenVolume` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
is enabled, the service account admission controller will
```

Reply:
done